tests/e2e/tests/test_redirects.py

```diff
@@ -150,20 +146,23 @@ def _verify_winxp_redirect_rules(self, base_url, product_alias, user_agent,):
         elif 'esr' in product_alias:
             assert '38.5.1esr.exe' in parsed_url.path
         elif product_alias in ['49.0b8', '49.0b10', '49.0b37']:
-            # beta versions are pinned to 49.0b10
-            assert '49.0b10.exe' in parsed_url.path
+            # beta versions are a moving number, regarless of which version is requested
+            # verify version is greater than or equal to 49
+            assert 49 < self._extract_windows_version_num(parsed_url.path)
```

Comment:
This still doesn't match the comment, as this would fail if the parsed version was 49.